### PR TITLE
Add Calendar page

### PR DESCRIPTION
**Pull request by encukou**
https://pyvo.cz/calendar/

### CODE REVIEW

**Comment by encukou** [comment]
cc @torsava

File: pyvocz/views.py [hunk 1]
```diff
@@ -163,7 +176,6 @@ def event(series_slug, date_slug):
     query = query.options(joinedload(tables.Event.talks))
     query = query.options(joinedload(tables.Event.venue))
     query = query.options(joinedload(tables.Event.talks, 'talk_speakers'))
-    query = query.options(joinedload(tables.Event.links))
```
Comment (encukou) [inline pyvocz/views.py]: Oops, that's a merge conflict :(
I'll add it back.

Comment (encukou) [inline pyvocz/views.py]: It's an optimization: the data is loaded in the initial SQL query, rather than when the attribute is first accessed.

Comment (torsava) [inline pyvocz/views.py]: Oups, yes, that's what I meant.

**Pull request merged**